.github/workflows/build.yml

```diff
@@ -86,3 +88,22 @@ jobs:
           if-no-files-found: ignore
           path: |
             /tmp/sphinx-*.log
+
+  release:
+    needs: [build]
+    if: startsWith(github.ref, 'refs/tags/v')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/download-artifact@v2
+        with:
+          name: jupyterlite dist ${{ github.run_number }}
+          path: ./dist
+      - name: Release
+        uses: softprops/action-gh-release@v1
+        with:
+          draft: true
+          prerelease: true
```

Comment:
Keeping `prerelease` for now.

Reply:
it's a bit of filagree, but CHANGELOG notes are nice... have done this regex-fu vs `---` ... though not sure if we _have_ one...

Reply:
hm... i don't see why it would get missed... no big, but worth exploring. might be because it doesn't have an extension? but its definitely the convention. at worst, we name it explicitly, as releases have no folders: 
```
dist/SHA256SUMS
dist/*
```